test/lit.cfg

```diff
@@ -211,6 +211,14 @@ for target in supported_targets:
     config.available_features.add(target)
   else:
     config.available_features.add('not-{}'.format(target))
+if 'WSL_DISTRO_NAME' in os.environ and 'WSL_INTEROP' not in os.environ:
```

Comment:
I think structuring it that way would be more readable but it's a matter of taste:
```Python
if 'WSL_DISTRO_NAME' in os.environ:
  if 'WSL_INTEROP' in os.environ:
    config.available_features.add('wsl-2')
  else:
    config.available_features.add('wsl-1')
else:
  config.available_features.add('not-wsl-1')
  config.available_features.add('not-wsl-2')
```

Reply:
Does the code in that way set 'not-wsl-1' on WSL 2 and 'not-wsl-2' on WSL 1 respectively?

Reply:
LOL - of course not. Somehow I thought we were filtering based on "wsl-1". Wasn't there an UNSUPPORTED keyword in lit? In short: it's fine.